### PR TITLE
Fix ad height

### PR DESCRIPTION
**Pull request by The-Vector-007**
aspect ratio padding trick

### CODE REVIEW

File: projects/commudle-admin/src/app/components/home/home.component.html [hunk 1]
```diff
@@ -12,7 +12,9 @@
         <app-home-external-feed-links></app-home-external-feed-links>
       </div>
       <div class="right-column">
-        <app-home-promotions></app-home-promotions>
+        <div class="wrapper" >
+          <app-home-promotions class="wrapper__img" ></app-home-promotions>
+        </div>
```
Comment (arpansac) [inline projects/commudle-admin/src/app/components/home/home.component.html]: rename class to: promotion or promotion_container

remove class from app-home-promotions

File: projects/commudle-admin/src/app/components/home/home.component.scss [hunk 2]
```diff
@@ -17,6 +17,21 @@
...
+          padding-top: calc(264 / 264 * 100%);
+        }
+        
+        .wrapper__img {
+          position: absolute;
+          top: 0;
+          left: 0;
+          max-width: 100%;
+          height: auto; 
+        }
```
Comment (arpansac) [inline projects/commudle-admin/src/app/components/home/home.component.scss]: move inside the body of above tag

Comment (arpansac) [reply projects/commudle-admin/src/app/components/home/home.component.scss]: In fact, optimize the solution

**Review by arpansac — CHANGES_REQUESTED** [review]
Just some mini changes

**Review by The-Vector-007 — COMMENTED** [review]
passed adHeight as well.

File: projects/commudle-admin/src/app/components/home/home.component.scss (hunk 2 above)
Comment (The-Vector-007) [reply projects/commudle-admin/src/app/components/home/home.component.scss]: Done!

File: projects/commudle-admin/src/app/components/home/home.component.html (hunk 1 above)
Comment (The-Vector-007) [reply projects/commudle-admin/src/app/components/home/home.component.html]: Done!

**Comment by Ashwin-op** [comment]
We also have to handle the edge case of no ad. That is if we set the adHeight of the nb-card in the page-ads.component.html and there's no ad available for a given slot then there'll be a blank white space. One possible solution could be to have a default ad for every slot so that no slot remains empty.

**Comment by arpansac** [comment]
Umm, let's skip it for now, we are in for a facelift soon!

**Pull request merged**